### DISCUSSION TITLE
Set up GitHub Sponsors?

### DISCUSSION THREAD

**Discussion by squidfunk** [ReactiveX/rxjs#7013] [Polls]
Have you considered accepting sponsorships? I bet there are many developers and companies that would like to contribute something to RxJS (or ReactiveX in general), me included, but there's currently no possibility, is there? RxJS is an amazing technology that should receive financial support.

**Comment by kwonoj** [comment]
Dupe of https://github.com/ReactiveX/rxjs/issues/6402 .

Core team discussed this matter in detail and so far our conclusion is we do not want external fund sponsoring. Things may change over time, but there are no changes in surroundings that we need to reevaluate this decision.

To double clarify, we welcome sponsoring in general if it meets the need of the team's work to move forward. We have few OSS sponsored CI / licenses, etcs. We do not set up direct money based sponsoring, however.

**Reply by squidfunk** [reply]
Thanks! I searched the issue tracker, but that didn't come up. Consider this question answered then. If you ever need funding, I'm happy to contribute.
